lib/devtools/rake/flay.rb

```diff
@@ -0,0 +1,110 @@
+module Devtools
+  module Rake
+    # Flay metric runner
+    class Flay
+      include Anima.new(:threshold, :total_score, :lib_dirs, :excludes), Procto.call(:verify), Adamantium
```

Comment:
When the include lines grow that long, I personally prefer to list each include in a separate line. Comma separated.